### DISCUSSION TITLE
Importing '@mantine/core/styles.css'; breaks css in React Vite application

### DISCUSSION THREAD

**Discussion by emaddeldin** [mantinedev/mantine#5702] [Q&A] [answered]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.5.1

### What package has an issue?

@mantine/core

### What framework do you use?

Vite

### In which browsers you can reproduce the issue?

All

### Describe the bug

I am building a dashboard for a client using React running on Vite. Using TailwindCSS only for styling. 
When I try to import the main stylesheet in App.jsx (@mantine/core/styles.css), the CSS changes on it's own. 
CSS changes include: white font turning black, navy blue background turning to sky blue, etc.

### Possible fix

Not sure yet, but it seems that the Mantine CSS is conflicting with my TailwindCSS, even if it's for basic HTML components.  
I hope there is a fix or solution for it, as this would force me to use another components library, this is my first time with Mantine btw.

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment, marked as answer]
Use one of the templates to get started, templates are tested to work correctly
